sage-ios/SAGE/SAGE/SignUpTableViewController.swift

```diff
@@ -74,7 +74,7 @@ class SignUpTableViewController: UITableViewController, UINavigationBarDelegate
     }
     
     func loadSchoolData() {
-        AnnouncementsOperations.loadSchools { (schoolDict) -> Void in
+        LoginHelper.loadSchools { (schoolDict) -> Void in
```

Comment:
@sameerav should i just take the whole thing out since theres no more loginhelper file